src/main/java/stsjorbsmod/cards/cull/Poultergeist.java

```diff
@@ -0,0 +1,41 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+
+public class Poultergeist extends CustomJorbsModCard {
```

Comment:
It looks like the card art is a derivative of a movie poster; we'd prefer to avoid stuff that's derivative of copyrighted works we don't have rights to redistribute, I'm afraid